<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="Celsius">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Celsius</string>
    </property>
    <widget class="QDial" name="CelsiusDial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="CelsiusLcd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Fahrenheit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>151</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Fahrenheit</string>
    </property>
    <widget class="QDial" name="FahrenheitDial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="FahrenheitLcd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>CelsiusDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CelsiusLcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FahrenheitDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FahrenheitLcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
